enterprise/server/backends/migration_cache/migration_cache_test.go

```diff
@@ -79,6 +79,84 @@ func (c *errorCache) Delete(ctx context.Context, d *repb.Digest) error {
 	return errors.New("error cache delete err")
 }
 
+func (c *errorCache) Contains(ctx context.Context, d *repb.Digest) (bool, error) {
+	return false, errors.New("error cache contains err")
+}
+
+func (c *errorCache) Metadata(ctx context.Context, d *repb.Digest) (*interfaces.CacheMetadata, error) {
+	return nil, errors.New("error cache metadata err")
+}
+
+func (c *errorCache) FindMissing(ctx context.Context, digests []*repb.Digest) ([]*repb.Digest, error) {
+	return nil, errors.New("error cache findmissing err")
+}
+
+func TestACIsolation(t *testing.T) {
+	te := testenv.GetTestEnv(t)
+	ctx := getAnonContext(t, te)
+	maxSizeBytes := int64(1000)
+	rootDirSrc := testfs.MakeTempDir(t)
+	rootDirDest := testfs.MakeTempDir(t)
+
+	srcCache, err := disk_cache.NewDiskCache(te, &disk_cache.Options{RootDirectory: rootDirSrc}, maxSizeBytes)
+	if err != nil {
```

Comment:
Please use require.NoError for all error checks. We've been a bit inconsistent throughout our tests but we should try to stick with require.NoError going forward

Reply:
Done